.github/workflows/client.yml

```diff
@@ -49,3 +49,17 @@ jobs:
         with:
           files: client/coverage/clover.xml
           flags: client-unit-integration-tests
+
+  publish-docker-image:
+    if: | 
+      github.event_name == 'push' &&
+      (startsWith(github.ref, 'refs/heads/feature/') || startsWith(github.ref, 'refs/heads/release-v'))
+      
+    name: Publish Docker image
+    needs: client
+    uses: INTO-CPS-Association/DTaaS/.github/workflows/docker.yml@feature/distributed-demo
+    with: 
+      registry: ghcr.io
+      image-name: into-cps-association/dtaas-client
+      version: ${{ github.sha }}
```

Comment:
### A quick idea:

What if the version number is obtained from package.json?

In client.yml and lib-ms.yml files,

apt-get install jq (or less preferably use [jq action](https://github.com/marketplace/actions/install-jq))

```txt
version=`cat package.json| jq -r '.version'`
```

The version variable needs to be created in the previous step. Github has variables for use across steps. Please see [this example](https://stackoverflow.com/questions/57819539/github-actions-how-to-share-a-calculated-value-between-job-steps).

Reply:
added the version